go/vt/vtctl/reparentutil/durability.go

```diff
@@ -61,8 +61,12 @@ func init() {
 
 // durabler is the interface which is used to get the promotion rules for candidates and the semi sync setup
 type durabler interface {
+	// promotionRule represents the precedence in which we want to tablets to be promoted.
+	// The lower the promotion rule of a tablet, the more we want it to be promoted in case of a failover
```

Comment:
Isn't this supposed to be higher, not lower?

Reply:
yes you are right, what was I thinking when I typed it

Reply:
fixed